MapboxStatic/SnapshotOptions.swift

```diff
@@ -0,0 +1,336 @@
+#if os(OSX)
+    import Cocoa
+#elseif os(watchOS)
+    import WatchKit
+#else
+    import UIKit
+#endif
+
+/**
+ A structure defining the viewpoint from which a snapshot is taken.
+ */
+@objc(MBSnapshotCamera)
+open class SnapshotCamera: NSObject {
+    /**
+     Vertical field of view, measured in degrees, for determining the altitude of the viewpoint.
+     */
+    static let angularFieldOfView: CLLocationDegrees = 30
+    
+    /**
+     Radius of the Earth, measured in meters.
+     */
+    static let radiusOfEarth: CLLocationDistance = 6_378_137
+    
+    /**
+     Size of a tile.
+     */
+    static let tileSize = CGSize(width: 512, height: 512)
```

Comment:
This is an private variable factored out for the purpose of keeping the altitude conversion code readable.